bitcoin/script/script.h

```diff
@@ -587,4 +573,28 @@ struct CScriptWitness
 
 bool CheckMinimalPush(const std::vector<unsigned char>& data, opcodetype opcode);
 
+/** Build a script by concatenating other scripts, or any argument accepted by CScript::operator<<. */
+template<typename... Ts>
+CScript BuildScript(Ts&&... inputs)
+{
+    CScript ret;
+    size_t cnt{0}; // Try to optimize over repeatedly calling ret.empty()
```

Comment:
Nit: also outdated now (feel free to ignore).

Reply:
Done